experiments/calibration/shared/rand_locations.jl

```diff
@@ -0,0 +1,39 @@
+using NCDatasets
+import EnsembleKalmanProcesses as EKP
+using ClimaUtilities.ClimaArtifacts
+import ClimaLand
+using StaticArrays
+
+# Returns a 2 SVectors of (lat, lon) tuples for n random locations on the land
+# surface. The two sets of locations are designed to be used as a training and
+# validation set.
+function rand_locations(surface_space, regridder_type, n = 100)
+    # Load the landsea mask data
+    datapath = ClimaLand.Artifacts.topmodel_data_path()
+    landsea_mask = SpaceVaryingInput(
+        datapath,
```

Comment:
Is `topmodel_data_path` the land sea mask?

Reply:
Right now CL doesnt really use a land sea mask except to set soil parameters over the ocean. We use a mask from a soil param dataset for that. In the future we could use the actual Clima land sea mask, but in the future we shouldnt have to do this. 

Otherwise,  in plotting, in computing bias and gloal rmse through ClimaAnalysis (mask defined in CA).

That said, here, we should probably be using the Clima land sea mask. Once Charlie's PR is merged and we stop doing computations over the ocean and need a mask to make the land domain, we can change everything to be etopo2022.